zbus/src/fdo/properties.rs

```diff
@@ -62,7 +62,7 @@ impl Properties {
         &self,
         interface_name: InterfaceName<'_>,
         property_name: &str,
-        value: Value<'_>,
+        value: &Value<'_>,
```

Comment:
Unfortunately, this will be a breaking change since this is a public API. :(

Reply:
Hmm, ok, lmk if / how you want to deal with it and I can update the patch. Alternatively feel free to just close this (or tag it as deferred or whatever you want :))